src/react-imgix.js

```diff
@@ -75,10 +77,14 @@ function buildSrc({
       srcSet = `${dpr2} 2x, ${dpr3} 3x`;
     } else {
       const buildSrcSetPair = targetWidth => {
-        const url = constructUrl(rawSrc, {
+        let urlParams = {
           ...srcOptions,
           width: targetWidth
-        });
+        };
+        if (!srcOptions.height && aspectRatio && aspectRatio > 0) {
+          urlParams.height = Math.floor(targetWidth / aspectRatio);
```

Comment:
@maxkolyanov is there any particular reason why Math.floor is used here over Math.round?

Reply:
Good catch, no particular reason here. Quite possibly that `Math.round()` will result in less incorrect outputs.